src/test/java/org/nebula/example/FeaturesPageObject.java

```diff
@@ -0,0 +1,21 @@
+package org.nebula.example;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class FeaturesPageObject extends BasePageObject {
+
+    @FindBy(css = "div#features>div.fixed-nav__positioning")
```

Comment:
idk the UI, check if `div.fixed-nav__positioning` is maybe enough and we do not have to go through parent